packages/content/content/docs/Ecosystem/LLMStyleGuide.mdx

```diff
@@ -1,16 +1,22 @@
-# Squiggle Style Guide
+---
+title: Style Guide
+---
+
+This is a style guide for how to write good Squiggle code. It's mostly designed to help LLMs write good Squiggle code, but could be useful for humans as well.
+
+---
```

Comment:
The prefix before `---` here and in `BasicPrompt.mdx` will be cut off in LLM system prompt; see `content/src/prompts.ts`.

This is a bit of an action-at-distance that I dislike because it seems fragile, but at least we check that `---` is present in `prompts.ts`, so there's _some_ safety net.